spec/models/job_spec.rb

```diff
@@ -243,10 +243,10 @@
           assert_queue_message
         end
 
-        # it "should deliver message without error" do
-        #   Host.any_instance.stub(:miq_proxy).and_return(@miq_server)
-        #   lambda { MiqQueue.first.deliver }.should_not raise_error
-        # end
+        it "should deliver message without error" do
+          allow_any_instance_of(Host).to receive(:miq_proxy).and_return(@miq_server)
```

Comment:
Is this actually important (given that it was commented out) and/or must we use `allow_any_instance_of`? (Might be the case, but thought I'd pose the question anyway)